docs/manual/constant-value-checker.tex

```diff
@@ -251,6 +251,39 @@ \section{Warnings\label{value-checker-warnings}}
 
 \end{itemize}
 \end{sloppypar}
+
+
+\section{Overflow\label{value-checker-overflow}}
+
+The Constant Value Checker normally takes Java's normal overflow rules into account when
+computing the possible values of expressions.
+The \code{-AignoreRangeOverflow} command-line option
+makes it ignore the possibility of overflow
+for range annotations (\<@IntRange>, for instance), making them more precise.
+You can supply the \code{-AignoreRangeOverflow} command-line option
+when you are confident that no overflow will occur in the program (for
```

Comment:
Thinking about this more, I suppose it is a convenience thing in the end; if the programmer reasons "I'm confident this isn't going to go above 10,000 or so, but I'm not sure exactly what the upper bound is exactly" (which is the sort of thing I'm thinking of), then they _could_ do the extra work to figure out what the bound exactly is, or they could instead use `-AignoreRangeOverflow`. I care more about the second argument that I made above, though - there's not any particular reason to couple overflow checking with some other checker (the value checker or the index checker or any other checker that doesn't explicitly check overflow). We don't try to reason about nullness in the value checker. Why would we try to reason about overflow?